### PR TITLE
Update Clerk as OAuth2 Provider with initial FAQ section

### PR DESCRIPTION
**Pull request by mzhong9723**

### CODE REVIEW

File: docs/advanced-usage/clerk-idp.mdx [hunk 1]
```diff
@@ -95,3 +95,9 @@ Below is an example of a response from `/oauth/userinfo`.
 }
 ```
 </Steps>
+
+## Frequently asked questions (FAQ)
+
+**When do the tokens expire?**
```
Comment (S3Prototype) [inline docs/advanced-usage/clerk-idp.mdx:101]:
```suggestion
### When do the tokens expire?**
```

**Comment by alexisintech** [comment]
What incited these changes?

**Comment by mzhong9723** [comment]
> What incited these changes?

@alexisintech Customer asked about token expirations for this feature, and I thought it should be something that's documented! https://clerkinc.slack.com/archives/C05LHV6AK50/p1708725371925889?thread_ts=1707062710.007209&cid=C05LHV6AK50

**Comment by alexisintech** [comment]
> > What incited these changes?
> 
> @alexisintech Customer asked about token expirations for this feature, and I thought it should be something that's documented! https://clerkinc.slack.com/archives/C05LHV6AK50/p1708725371925889?thread_ts=1707062710.007209&cid=C05LHV6AK50

awesome ! just wanted that info in here, thank uuu

**Pull request merged**